backend/src/Manager/AdminManager.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Manager\Admin;
+namespace App\Manager;
```

Comment:
Why moved AdminManager to be under Manager layer directly?

Reply:
done